asf_welsh_energy_consultation/analysis/produce_plots_and_stats.py

```diff
@@ -113,15 +113,15 @@
         alt.Chart(
             new_build_hp_cumulative,
             title=[
-                "Cumulative total of EPC registrations for new builds",
-                "with heat pumps in Wales since 2008",
+                "Cumulative heat pump installations in new builds with EPC registrations in Wales since 2008",
```

Comment:
```suggestion
                "Cumulative heat pump installations in new builds in Wales since 2008",
```
I think you can have a simpler title and have two notes in the report to say:

1. these are MCS certified installations;
2. that we're identifying new builds through enhancing MCS with EPC through fuzzy matching on address data.

Reply:
Great idea, much cleaner.